api/go.sum

```diff
@@ -44,8 +44,6 @@ github.com/google/go-cmp v0.4.0/go.mod h1:v8dTdLbMG2kIc/vJvl+f65V22dbkXbowE6jgT/
 github.com/google/go-cmp v0.5.9 h1:O2Tfq5qg4qc4AmwVlvv0oLiVAGB7enBSJ2x2DqQFi38=
 github.com/google/go-cmp v0.5.9/go.mod h1:17dUlkBOakJ0+DkrSSNjCkIjxS6bF9zb3elmeNGIjoY=
 github.com/google/gofuzz v1.0.0/go.mod h1:dBl0BpW6vV/+mYPU4Po3pmUjxk6FQPldtuIdl/M65Eg=
-github.com/hashicorp/consul/proto-public v0.5.4-rc1 h1:psIJ9DgJw5QbRsh6RGYYr3Viw+r+up9PE7Z6mAZsAEQ=
-github.com/hashicorp/consul/proto-public v0.5.4-rc1/go.mod h1:a1pOtKbQ2+iRnMlEA2bywlEZ0nbCQ2pS7GDQN6pqLwU=
```

Comment:
I believe these were incorrectly included in the RC branch, which `release/1.20.0` originated from. Checking other releases, these `go.sum` entries **shouldn't** be present and the `go.mod` replacements **should** be present.

You'll see similar cases in the `go.sum` files below